dev-tools/compose/docker-compose.discovery.prod.yml

```diff
@@ -150,6 +150,11 @@ services:
       - eth-contracts-abis:/audius-discovery-provider/build/eth-contracts
       - solana-programs-idl:/audius-discovery-provider/idl
       - ${PROJECT_ROOT}/dev-tools:/tmp/dev-tools
+    healthcheck:
+      test: [
+          "CMD-SHELL",
+          "pgrep pg_migrate || curl -f http://localhost:5000/health_check?bypass_errors=true || exit 1"
```

Comment:
maybe in this test scenario we want to fail health check if there are errors, so removing bypass_errors